src/main/java/com/jcabi/log/Logger.java

```diff
@@ -429,6 +429,32 @@ public void write(final int data) throws IOException {
     }
 
     /**
+     * Log messages constructed from Suppliers (Java 8 users).
+     * It is convenient to use method referencing because the method
```

Comment:
@amihaiemil I don't think it's just about convenience. We need to highlight that it is for slow and expensive calls